kaitlyn-mcgoldrick/my-app/src/App.js

```
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import './App.css';
+import { Header } from './Components/header';
+import { PokeCard } from './Components/pokecard';
+import Form from './Components/form';
+const fetch = require('node-fetch');
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      picture: '',
+      error: ''
+    }
+    this.handleFormSubmit = this.handleFormSubmit.bind(this);
+  }
+
+  handleFormSubmit = (name) => {
+    this.fetchPokemon(name);
+  }
+
+  fetchPokemon = (idOrName) => {
+    fetch(`https://pokeapi.co/api/v2/pokemon/${idOrName}`)
+    .then(res => {
+      console.log(res.status);
+      if (res.status >= 200 && res.status < 400) {
+        return res;
+    } else {
+      var err = new Error("We can't seem to find a pokemon with that name or number.")
```

For the sake of writing code in an "immutable way", every `var` should be replaced with `const` (ideal) or a `let` (only if you really have to). This one can be switched to a `const`! :)

---

or in this case you can just return the new Error without creating a new variable `return new Error(...)`